piet-svg/examples/demo.rs

```diff
@@ -0,0 +1,18 @@
+//! Basic example of rendering to a SVG
```

Comment:
Naming doesn't follow the pattern, this should be "basic-svg.rs". That said, I agree the naming scheme is not ideal, and the d2d one also doesn't follow the pattern.